README.md

```diff
@@ -1,10 +1,87 @@
+| [IDR repository](https://github.com/IDR/idr0133-dahlin-cellpainting/tree/main/screenA) | Repository containing annotated screen data |
+| [Supplemental metadata](https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-023-36829-x/MediaObjects/41467_2023_36829_MOESM5_ESM.xlsx) | Supplementary metadata dataset |
+| [JUMP](https://jump-cellpainting.broadinstitute.org/) | JUMP data repository|
+
+## Repository structure
+
+Overall structure of our repo.
+
+| Directory | Description |
+|-----------|-------------|
+| [Data](./data) | Contains all the datasets |
+| [Results](./results) | Stores all results generated from the notebook modules in the `./notebooks` directory |
+| [Src](./src) | Contains utility functions |
+| [Notebooks](./notebooks/) | Contains all notebook modules for our analysis |
+
+## Notebook structure
+
+Below are all the notebook modules used in our study.
+
+| Notebook | Description |
+|----------|-------------|
+| [0.feature_selection](./notebooks/0.feature_selection/) | Labels wells with cellular injuries, identifies shared features between the cell injury dataset and JUMP, and selects features for the cell injury dataset |
+| [1.data_splits](./notebooks/1.data_splits/) | Generates training and testing splits, including holdout sets (plate, treatment, and random wells) |
+| [2.modeling](./notebooks/2.modeling/) | Trains and evaluates a multi-class logistic regression model |
+| [3.jump_analysis](./notebooks/3.jump_analysis/) | Applies our model to the JUMP dataset to predict cellular injuries |
+| [4.visualizations](./notebooks/4.visualizations/) | Contains a notebook responsible for generating our figures |
+
+## Analysis summary
+
+### Feature selection
+
+Before conducting any feature selection processes, we first labeled wells associated with an injury.
+We achieved this using the datasets downloaded from the [cell-injury](https://www.nature.com/articles/s41467-023-36829-x) study, which provided information on which treatments were associated with which injuries.
+After mapping injury labels onto the wells based on their treatments, we applied feature alignment.
+
+We identified which features in the `cell-injury` dataset were present in the JUMP dataset.
+Once identified, we used only the "shared" features—morphological features common to both the JUMP and cell-injury datasets.
+
+Next, we applied feature selection using [Pycytominer](https://github.com/cytomining/pycytominer) to obtain informative features.
+This process generated our `feature-selected` profiles, which will be used to train our multi-class logistic regression model.
+
+### Data splitting
+
+We split our data into training, testing, and holdout sets.
+First, we generated our holdout splits, which are reserved exclusively for evaluating the model's performance and are not used during the model's training or tuning.
+
+1. **Plate Holdout Set**: We randomly selected 10 plates.
+2. **Treatment Holdout Set**: We selected one treatment to hold out.
```

Comment:
This might be confusing. Did you hold out only one treatment or was it one treatment per MoA??

Reply:
Or not MoA but injury type in this case